### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pietgeursen**

### CODE REVIEW

**Comment by pietgeursen** [comment]
Actually, just forked this and making a slush-pages-react repo that has es-2015 and react set up.

**Comment by ahdinosaur** [comment]
awesome! this makes me so happy. :smiley:

File: template/package.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
   "description": "<%= description %>",
   "main": "index.js",
   "scripts": {
-    "start": "budo . -d --serve bundle.js --live",
-    "build": "browserify . -o bundle.js -g uglifyify",
-    "test": "browserify test.js | smokestack | tap-spec",
+    "start": "budo . -t babelify -d --serve bundle.js --live",
```
Comment (ahdinosaur) [inline template/package.json:7]: you should only add `babelify` to the `browserify.transform` options within the rest of your `package.json`, as so:

``` json
{
  ...
  "browserify": {
    "transform": [
      "babelify"
    ]
  },
  ...
}
```

this way you keep your config DRY.

**Comment by pietgeursen** [comment]
Thanks Mikey!

**Pull request closed without merging**